src/apps/openfluid-devstudio/MainWindow.cpp

```diff
@@ -456,87 +526,114 @@ void MainWindow::createMenus()
 // =====================================================================
 
 
-void MainWindow::setWorkspaceDefaults()
+void MainWindow::updateBuildJobsStatus()
 {
-  auto* WMgr = openfluid::base::WorkspaceManager::instance();
+  QString JobStatusString;
+  if (m_WareBuildOptions.m_IsParallelJobs)
+  {
+    JobStatusString = QString::fromStdString(std::to_string(m_WareBuildOptions.m_JobsNumber))+" "+tr("jobs");
+    mp_WidgetsCollection->setBuildJobs(m_WareBuildOptions.m_JobsNumber);
+  }
+  else
+  {
+    JobStatusString = tr("No parallel jobs");
+    mp_WidgetsCollection->setBuildJobs(0); // NOT SURE
+  }
+  mp_BuildStatusWidget->getLabel("Jobs")->setText(JobStatusString);
+}
 
-  WMgr->openWorkspace(openfluid::base::PreferencesManager::instance()->getCurrentWorkspacePath());
 
+// =====================================================================
+// =====================================================================
 
-  std::string ConfigMode = WMgr->getWaresConfigureMode();
-  std::string BuildMode = WMgr->getWaresBuildMode();
 
-  mp_Toolbar->optionsWidget()->setConfigureMode((ConfigMode == "DEBUG") ?
-                                                openfluid::waresdev::WareSrcContainer::ConfigMode::CONFIG_DEBUG :
-                                                openfluid::waresdev::WareSrcContainer::ConfigMode::CONFIG_RELEASE);
+void MainWindow::updateBuildModeStatus()
+{
+  if (m_WareBuildOptions.m_IsWithInstall)
+  {
+    mp_BuildStatusWidget->getLabel("Installed")->setText(tr("with install"));
+  }
+  else
+  {
+    mp_BuildStatusWidget->getLabel("Installed")->setText(tr("without install"));
+  }
+  mp_WidgetsCollection->setBuildMode(m_WareBuildOptions.getBuildMode());
+}
 
-  mp_Toolbar->optionsWidget()->setBuildMode((BuildMode == "BUILDONLY") ?
-                                            openfluid::waresdev::WareSrcContainer::BuildMode::BUILD_NOINSTALL :
-                                            openfluid::waresdev::WareSrcContainer::BuildMode::BUILD_WITHINSTALL);
 
+// =====================================================================
+// =====================================================================
 
-  std::vector<std::string> LastOpenWares = WMgr->getOpenWaresPaths();
 
-  for (const auto& WarePath : LastOpenWares)
+void MainWindow::updateConfigureModeStatus()
+{
+  if (m_WareBuildOptions.m_IsReleaseMode)
   {
-    mp_WidgetsCollection->openPath(QString::fromStdString(WarePath));
+    mp_BuildStatusWidget->getLabel("Mode")->setText(QString::fromStdString("release"));
```

Comment:
use tr() for later translation
the conversion  back and forth to std::string seems unnecessary